lib/auth/SpacedeckAuthProvider.js

```diff
@@ -47,26 +54,24 @@ class SpacedeckAuthProvider {
     }
 
     async signout() {
-        const response = await fetch(this.url + '/api/sessions/current', {
+        return await fetch(this.url + '/api/sessions/current', {
```

Comment:
Never `return await`, only `return`. And then the consumer of this method gets to decide if they want to wait for the process to finish or not.